src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -200,21 +200,59 @@ public async Task<List<UserViewModel>> ListPreReleaseUsersAsync()
                 .CheckCanManageAllUsers()
                 .OnSuccess(async () =>
-                    await _usersAndRolesDbContext.UserInvites
-                        .AsQueryable()
-                        .Where(ui => !ui.Accepted)
-                        .OrderBy(ui => ui.Email)
-                        .Include(ui => ui.Role)
-                        .Select(ui => new UserViewModel
+                    {
+                        var pendingInvites = await _usersAndRolesDbContext.UserInvites
+                            .AsQueryable()
+                            .Where(ui => !ui.Accepted)
+                            .OrderBy(ui => ui.Email)
+                            .Include(ui => ui.Role)
+                            .Select(ui => new PendingInviteViewModel
+                            {
+                                Email = ui.Email,
+                                Role = ui.Role.Name,
+                            }).ToListAsync();
+
+                        foreach (var invite in pendingInvites)
                         {
-                            Email = ui.Email,
-                            Role = ui.Role.Name,
-                        }).ToListAsync()
+                            var userReleaseInvites = await _contentDbContext.UserReleaseInvites
+                                .Include(userReleaseInvite => userReleaseInvite.Release.Publication)
+                                .Where(userReleaseInvite =>
+                                    userReleaseInvite.Email.ToLower() == invite.Email.ToLower())
+                                .ToListAsync();
+
+                            invite.UserReleaseRoles = userReleaseInvites
+                                .Select(userReleaseInvite =>
+                                    new UserReleaseRoleViewModel
+                                    {
+                                        Publication = userReleaseInvite.Release.Publication.Title,
+                                        Release = userReleaseInvite.Release.Title,
+                                        Role = userReleaseInvite.Role,
+                                    }
+                                ).ToList();
+
+                            var userPublicationInvites = await _contentDbContext.UserPublicationInvites
+                                .Include(userPublicationInvite => userPublicationInvite.Publication)
+                                .Where(userPublicationInvite =>
+                                    userPublicationInvite.Email.ToLower() == invite.Email.ToLower())
+                                .ToListAsync();
+
+                            invite.UserPublicationRoles = userPublicationInvites
+                                .Select(userPublicationInvite =>
+                                    new UserPublicationRoleViewModel
+                                    {
+                                        Publication = userPublicationInvite.Publication.Title,
```

Comment:
As mentioned in the frontend, we should map the `Id` here so that we can use it in `key` props in the table rows